### PR TITLE
Release notes for Percona Server 5.5.50-38.0

### PR DESCRIPTION
**Pull request by hrvojem**
Upstream bugs fixed list update
Fixed LP Bug #1505255

### CODE REVIEW

File: doc/source/flexibility/extended_tls_support.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. _extended_tls_support:
+
+=============================
+Support for TLS v1.1 and v1.2
+=============================
+
+|Percona Server| has implemented TLS v1.1 and v1.2 protocol support and at the same time disabled TLS v1.0 support (support for TLS v1.0 can be enabled by adding the ``TLSv1.0`` to :variable:`tls_version` variable). Support for TLS v1.1 and v1.2 protocols has been implemented by porting the :variable:`tls_version` variable from 5.7 server. TLS v1.0 protocol has been disabled because it will no longer be viable for `PCI after June 30th 2016 <https://www.pcisecuritystandards.org/documents/Migrating_from_SSL_Early_TLS_Information%20Supplement_v1.pdf>`_. Variable default has been changed from ``TLSv1,TLSv1.1,TLSv1.2`` to ``TLSv1.1,TLSv1.2`` to disable the support for TLS v1.0 by default.
```
Comment (percona-ysorokin) [inline doc/source/flexibility/extended_tls_support.rst]: `TLSv1.0` vs `TLSv1` discrepancies for the value of `tls_version`

File: doc/source/release-notes/Percona-Server-5.5.50-38.0.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.50-38.0 on July 8th, 2016. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.50-38.0/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.50 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-50.html>`_, including all the bug fixes in it, |Percona Server| 5.5.50-38.0 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.50-38.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.50-38.0>`_. 
+
+New Features
+============
+
+ |Percona Server| has implemented protocol support for :ref:`TLS 1.1 and TLS 1.2 <extended_tls_support>`. This implementation automatically disables TLS v1.0 support.
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.5.50-38.0.rst]: `This implementation automatically disables TLS v1.0 support` sounds too strong. I would say something like `turns this protocol off by default`.

File: doc/source/release-notes/Percona-Server-5.5.50-38.0.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ Fixed memory leak in ``mysqlbinlog``. Bug fixed :bug:`1582761` (upstream :mysqlbug:`78223`).
+
+ Fixed memory leaks in ``mysqldump``. Bug fixed :bug:`1587873` and :bug:`1588845` (upstream :mysqlbug:`81714`).
+
+ Fixed memory leak in ``innochecksum``. Bug fixed :bug:`1587873`.
+
+ Fixed memory leak in non-existing defaults file handling. Bug fixed :bug:`1588344`.
+
+ Fixed memory leak in ``mysqlslap``. Bug fixed :bug:`1587873`.
+
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.5.50-38.0.rst]: Please, also mention lp:1596845 / http://bugs.mysql.com/bug.php?id=82026 (Stack buffer overflow with
--ssl-cipher=<more than 4K characters>).
This is from the updated commit message https://github.com/percona/percona-server/commit/deddfcadb70663f32fb71e1737a3cf512a390314

Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.5.50-38.0.rst]: I see that https://bugs.launchpad.net/percona-server/+bug/1596845 is marked invalid for 5.5, should the milestone be updated as well for 5.5 then?

**Comment by percona-ysorokin** [comment]
LGTM

**Pull request merged**